include/glow/Runtime/Host Manager/HostManager.h

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Is there a space in the path name?

Reply:
Maybe we can make it one word, like runtime or ExecutionEngine?

Reply:
Agreed, space removed.